src/lib/viewers/controls/icons/IconArrowLeft24.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+
+function IconArrowLeft24(props: React.SVGProps<SVGSVGElement>): JSX.Element {
```

Comment:
Could we break some of these changes out into separate PRs? It will speed up the review process.